app/src/main/java/ch/epfl/skysync/models/user/Admin.kt

```diff
@@ -0,0 +1,23 @@
+package ch.epfl.skysync.models.user
+
+import ch.epfl.skysync.models.calendar.AvailabilityCalendar
+import ch.epfl.skysync.models.calendar.FlightGroupCalendar
+import ch.epfl.skysync.models.flight.RoleType
+
+class Admin(
+    override val firstname: String,
+    override val lastname: String,
+    override val id: String,
+    override val availabilities: AvailabilityCalendar,
+    override val assignedFlights: FlightGroupCalendar,
+) : User {
+  private val roleTypes: Set<RoleType> = setOf()
```

Comment:
I don't really get why `roleTypes` is not part of the `Flight` interface, but if you make the `roleTypes` arrtibute `private` and don't implement the `addRoleType` method, the database side has no way of populating this attribute. (We can leave it as it is for now and change it later, we'll just have to keep it in mind)